server/tests/util/mock.ts

```diff
@@ -0,0 +1,29 @@
+import { collections, dbClient } from '../../db';
+import { EmployerSiteBatchSchema, validate } from '../../validators';
+
+/**
+ * Function to save multiple sites worth of mock data.
+ * NOTE: this should probably be replaced with existing logic from in-use services as some point.
+ * @param sitesArg Site or array of sites to save
+ * @returns Array of responses
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const saveSites = async (sitesArg) => {
```

Comment:
How much work would it be to use `saveSingleSite` to create the mocks and remove this function?

Reply:
Refactored in latest commit!